fsmonitor.c

```diff
@@ -244,14 +258,16 @@ void add_fsmonitor(struct index_state *istate)
 		/* Update the fsmonitor state */
 		refresh_fsmonitor(istate);
 	}
+
+	strbuf_release(&last_update);
```

Comment:
This is not necessary, as `strbuf_detach()` already re-sets `last_update` to no longer point to any allocated memory.